Makefile

```diff
@@ -128,8 +128,7 @@ shellcheck:
 
 tmp/rules.yaml: $(GOJSONTOYAML_BIN) $(ASSETS)
 	mkdir -p tmp/rules
-	for f in $(shell find assets/ -name '*prometheus-rule.yaml'); do $(GOJSONTOYAML_BIN) -yamltojson < "$$f" | jq .spec > "tmp/rules/$$(echo "$$f" | sed 's/\//-/g').json"; done
-	jq -s '[.[]] | { groups: map(.groups[]) }' $$(ls -d tmp/rules/*) | $(GOJSONTOYAML_BIN) > tmp/rules.yaml
+	hack/find-rules.sh | $(GOJSONTOYAML_BIN) > tmp/rules.yaml
```

Comment:
This won't work as `groups` need to be joined together. This is why unit tests are failing.

Reply:
Ah right, thanks. Done.